apps/server/src/modules/board/controller/mapper/submission-container-content-element-response.mapper.ts

```diff
@@ -0,0 +1,30 @@
+import { ContentElementType, SubmissionContainerElement } from '@shared/domain';
+import { SubmissionContainerElementContent, SubmissionContainerElementResponse, TimestampsResponse } from '../dto';
+import { BaseResponseMapper } from './base-mapper.interface';
+
+export class SubmissionContentElementResponseMapper implements BaseResponseMapper {
```

Comment:
file name is ok, but class name should be 
SubmissionContainerElementResponseMapper